cmd/validation/main.go

```diff
@@ -6,14 +11,25 @@ import (
 )
 
 func main() {
```

Comment:
Do we need to add a lot of comments? It's good; it's self-explanatory.

Reply:
@TingSyuanWang Not sure about your idea, but I think the comments make this simple code harder to understand.

Reply:
@maxsarratt Adding excessive comments to parts of the code that do not require special explanations can be unnecessary. The code itself may already be self-explanatory and easily understood. Thus, I concur with your viewpoint that it is advisable to refrain from adding comments in such cases.